micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSView.js

```
@@ -125,30 +117,110 @@
-    const {action:actionSelected} = action 
-    //action===EDIT go to edit screen 
-    if(actionSelected==="EDIT") {
+    const { action: actionSelected } = action
+    if (actionSelected === "EDIT") {
       const additionalParamString = new URLSearchParams(additionalParams).toString();
-      history.push(`/${window?.contextPath}/employee/workbench/mdms-edit?moduleName=${moduleName}&masterName=${masterName}&uniqueIdentifier=${uniqueIdentifier}${additionalParamString ? "&"+additionalParamString : ""}`)
-    }
-    //action===DISABLE || ENABLE call update api and show toast respectively
-    else{
-      //call update mutation
+      history.push(`/${window?.contextPath}/employee/workbench/mdms-edit?moduleName=${moduleName}&masterName=${masterName}&uniqueIdentifier=${uniqueIdentifier}${additionalParamString ? "&" + additionalParamString : ""}`)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)